ley.config.mjs

```diff
@@ -1,13 +1,42 @@
-const options = {};
+import fs from 'node:fs';
+import dotenv from 'dotenv';
 
-if (process.env.POSTGRES_URL) {
-  options.ssl = true;
+// End replacement for dotenv-safe
+export function config() {
+  if (process.env.FLY_IO) return;
 
-  // Set standard environment variables
-  process.env.PGHOST = process.env.POSTGRES_HOST;
-  process.env.PGDATABASE = process.env.POSTGRES_DATABASE;
-  process.env.PGUSERNAME = process.env.POSTGRES_USER;
-  process.env.PGPASSWORD = process.env.POSTGRES_PASSWORD;
+  dotenv.config();
+
+  // Replacement for unmaintained dotenv-safe package
+  // https://github.com/rolodato/dotenv-safe/issues/128#issuecomment-1383176751
+  //
+  // TODO: Remove this and switch to dotenv/safe if this proposal gets implemented:
+  // https://github.com/motdotla/dotenv/issues/709
+  const unconfiguredEnvVars = Object.keys(
+    dotenv.parse(fs.readFileSync('./.env.example')),
+  ).filter((exampleKey) => !process.env[exampleKey]);
+
+  if (unconfiguredEnvVars.length > 0) {
+    throw new Error(
+      `.env.example environment ${
+        unconfiguredEnvVars.length > 1 ? 'variables' : 'variable'
+      } ${unconfiguredEnvVars.join(', ')} not configured in .env file`,
+    );
+  }
```

Comment:
I guess this will throw on Vercel deploy (because the `.env` file won't exist and some environment variables will be unconfigured)

Reply:
9fa36ec